llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1154,6 +1154,22 @@ Instruction *SPIRVToLLVM::transCmpInst(SPIRVValue *bv, BasicBlock *bb, Function
   return inst;
 }
 
+// =====================================================================================================================
+// Perform transposition on given input type.
```

Comment:
Provide a bit more context:
```suggestion
// Perform matrix transpose on given input type.
```